lms/static/js/spec/main.js

```diff
@@ -514,6 +514,34 @@
             'annotator': {
                 exports: 'Annotator',
                 deps: ['jquery']
+            },
+            'js/search/views/form': {
```

Comment:
Is there a reason that you're using the Backbone shim config to declare these dependencies and such, instead of using the `define` function in the Javascript files so that require.js picks them up automatically?